tree/dataframe/src/RCsvDS.cxx

```diff
@@ -465,7 +465,7 @@ std::vector<std::pair<ULong64_t, ULong64_t>> RCsvDS::GetEntryRanges()
 
    const auto chunkSize = nRecords / fNSlots;
    const auto remainder = 1U == fNSlots ? 0 : nRecords % fNSlots;
-   auto start = 0ULL == fEntryRangesRequested ? 0ULL : fProcessedLines;
```

Comment:
What's the reason for this change?

Reply:
you might also ask what was the reason for the previous version...I couldn't find any, to me it seems that if `fEntryRangesRequested` is 0 then `fProcessedLines` must be 0 too

Reply:
But the previous logic relied on the value of `fEntryRangesRequested` to assign `start` to either `0` or `fProcessLines`, and the new logic always assigns to `fProcessedLines`. You're saying this change can be safely made?

Reply:
yes, see my previous message, but please do confirm that's the case :)